docs/pages/admin-guides/teleport-policy/integrations/azure-sync.mdx

```diff
@@ -96,6 +108,24 @@ discovery_service:
 
 The Discovery Service will now periodically fetch resources from your Azure subscription.
 
+### Activating the Discovery Service
+
+To activate the Discovery Service for fetching Azure resources, you'll need to authorize the identity running the
+Discovery service. Two options for authorization exist:
+
+1. [Managed identity attached to a VM](https://goteleport.com/docs/enroll-resources/auto-discovery/servers/azure-discovery/#attach-an-identity-to-your-azure-vm) - Use the managed identity ID as the principal ID in step 2 of integration command
```

Comment:
doc links should be relative. otherwise they're not specific to the version but will go to the current version.